src/Microsoft.ServiceFabric.Services/TelemetryConstants.cs

```diff
@@ -0,0 +1,59 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Services
+{
+    internal static class TelemetryConstants
+    {
+        // Names of all the telemetry events.
+        internal static readonly string ServiceLifecycleEventName = "TelemetryEvents.ServiceLifecycleEvent";
+        internal static readonly string ActorStateProviderUsageEventName = "TelemetryEvents.ActorStateProviderUsageEvent";
```

Comment:
>TelemetryEvents [](start = 76, length = 15)

Please move Actor related constants to Actors project.

Reply:
Thanks, have moved those constants.